src/core/onerror.js

```diff
@@ -1,43 +1,23 @@
-import { window } from "../globals";
 import { pushFailure, test } from "../test";
 
 import config from "./config";
-import { defined, extend } from "./utilities";
+import { extend } from "./utilities";
 
-( function() {
-	if ( !defined.document ) {
-		return;
-	}
-
-	// `onErrorFnPrev` initialized at top of scope
-	// Preserve other handlers
-	var onErrorFnPrev = window.onerror;
-
-	// Cover uncaught exceptions
-	// Returning true will suppress the default browser handler,
-	// returning false will let it run.
-	window.onerror = function( error, filePath, linerNr ) {
-		var ret = false;
-		if ( onErrorFnPrev ) {
-			ret = onErrorFnPrev( error, filePath, linerNr );
-		}
-
-		// Treat return value as window.onerror itself does,
-		// Only do our handling if not suppressed.
-		if ( ret !== true ) {
-			if ( config.current ) {
-				if ( config.current.ignoreGlobalErrors ) {
-					return true;
-				}
-				pushFailure( error, filePath + ":" + linerNr );
-			} else {
-				test( "global failure", extend( function() {
-					pushFailure( error, filePath + ":" + linerNr );
-				}, { validTest: true } ) );
-			}
-			return false;
+// Handle an unhandled exception. By convention, returns true if further
+// error handling should be suppressed and false otherwise.
+// In this case, we will only suppress further error handling if the
+// "ignoreGlobalErrors" configuration option is enabled.
```

Comment:
I wish I knew! This configuration option was present in the original `window.onerror` that is currently on master.

From the looks of it, it seems to just suppress QUnit's reporting of any unhandled errors (i.e., errors that make it to window.onerror).

I could see an argument that perhaps this logic should be kept in `window.onerror` and that any call to `QUnit.onError` should report the error no matter what.